apps/vr-tests/.storybook/main.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 
     config.module.rules.unshift({
       test: /\.(ts|tsx)$/,
-      use: [{ loader: '@fluentui/make-styles-webpack-loader' }],
+      use: [{ loader: '@griffel/webpack-loader' }],
```

Comment:
nit: shouldn't webpack loader convention be like `griffel-loader` ?

Reply:
AFAIK there is no strict convention, other projects use similar naming.

https://github.com/atlassian-labs/compiled/blob/5a4494b1990c27390abfc29df7e47dc28e8d25dc/packages/webpack-loader/package.json#L2
https://github.com/callstack/linaria/blob/57b13ffe878d794ff8e3c38f944983432e754e27/packages/webpack-loader/package.json#L2
https://github.com/seek-oss/vanilla-extract/blob/2b0abcd646cbbc8836301822c27c10f393870f4a/packages/webpack-plugin/package.json#L2